public/src/admin/manage/category.js

```diff
@@ -64,7 +64,7 @@ define('admin/manage/category', [
 		}
 
 		
-		$('form.category input, form.category select')
+		$('form.category').find('input,textarea,select')
```

Comment:
>  is this necessary?

Yes, to minimize conflicts during **pull**.
![2015-09-17 22 38 56](https://cloud.githubusercontent.com/assets/465108/9943922/e601dc38-5d8c-11e5-9aa4-03725f295471.png)

:)